### PR TITLE
Ewb 1825 fix tests maybe

### PR DESCRIPTION
**Pull request by kgreav**

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-1825 Fix python SDK tests](https://app.clickup.com/t/6929263/EWB-1825)

**Comment by kgreav** [comment]
@charlta 
So with these changes (excluding database tests) everything works in tox but fails in the IDE.
Basically when run through tox all the fixtures are returning coroutines and need to be awaited (exactly once). On the other hand the IDE seems to be doing the awaiting for us, and thus they break in the IDE. This is also true if you run pytest by hand.

I assume tox is doing something weird, but I haven't figured out why yet - I assume maybe the containers are getting a new version of tox which is causing it.

Note: This PR should not be merged as I think it is wrong.

File: test/database/sqlite/test_database_sqlite.py [hunk 1]
```diff
@@ -417,35 +417,35 @@ def test_schema_power_electronics_wind_unit(self, caplog, data):
 
     # noinspection PyShadowingNames
     @log_on_failure_decorator
-    @settings(deadline=1000)
-    @given(data())
-    def test_schema_ac_line_segment(self, caplog, data):
-        self._validate_schema(SchemaNetworks().network_services_of(AcLineSegment, data.draw(create_ac_line_segment(False))))
+    @settings(deadline=2000)
+    @given(acls=create_ac_line_segment(False))
```
[the remark this answers is not in the document]
Comment (charlta) [reply test/database/sqlite/test_database_sqlite.py:421]: @kgreav If it works that is great. I suspect I was using `draw` as I did not know you could rename the stuff in `@given` and was trying to make it so I could pass in the `False`, and that was the online help I found.

**Comment by kgreav** [comment]
@charlta so I think the issue was a combination of things - not all things I can explain. There seems to be potentially a version issue with pytest-asyncio/pytest, where config from the tox.ini wasn't being picked up properly except for by tox. I don't know why this is only appearing now. Maybe Marcus got a newer version where asyncio-mode defaulted to strict which then confused us. 

This seems to do the right thing everywhere with this configuration. You'll have to `pip install -U '.[test]'` to make sure you have all the right versions.
My other change seems to have only made the database tests marginally faster, but it reads better anyway.

**Pull request merged**